Wireframe/index.html

```diff
@@ -1,13 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Wireframe</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <!-- Header section -->
+  <header class="site-header">
+    <h1>Wireframe Exercise</h1>
+    <p>Exploring Git Concepts</p>
+  </header>
+
+  <!-- Main content section -->
+  <main>
+    <section class="main-container">
+      <!-- Article with main image -->
+      <article>
+        <img src="images/git.jpg" alt="git photo" class="img-pic">
+        <div class="inner">
+          <h2>Git</h2>
+          <p>Git is a distributed version control system that allows developers to track changes in their codebase over time.</p>
+          <a href="https://en.wikipedia.org/wiki/Git" target="_blank" class="btn" style="margin-top: 9px;">Read more</a>
+        </div>
+      </article>
+    </section>
+
+    <!-- Article container section -->
+    <section class="article-container">
+      <!-- Article 1: Developers Need Git -->
+      <article class="article">
+        <img src="images/git-benefits.jpg" alt="git benefits photo" class="box">
+        <div class="inner">
+          <h4>Developers Need Git</h4>
+          <p>Developers use Git to collaborate on projects, track changes, and maintain a history of their work, enabling easy debugging and teamwork.</p>
+          <a href="https://docs.github.com/en/get-started/using-git/about-git" target="_blank" class="btn" style="margin-top: 19px;">Read more</a>
+        </div>
+      </article>
+
+      <!-- Article 2: Branch in Git -->
+      <article class="article">
+        <img src="images/git-branch.jpg" alt="Git branch photo" class="box">
+        <div class="inner">
+          <h4>Branch in Git</h4>
+          <p>In Git, a branch is a new/separate version of the main repository.</p>
+          <a href="https://www.w3schools.com/git/git_branch.asp?remote=github" target="_blank" class="btn" style="margin-top: 45px;">Read more</a>
```

Comment:
Why have you put inline styles on these buttons but not anywhere else? What problem are you solving with this strategy?

Reply:
this anchor button is closer to the text and I didn't want it to look like this. I'm trying to organize the anchor buttons to all look organized to the left bottom. I added the inline style to push down this button to the bottom making all the anchor  buttons organized :grin: